src/test/Missions/Mission.test.jsx

```diff
@@ -0,0 +1,48 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
- [ ] Good job on adding tests. Kindly make sure to add tests for all your components and functions as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20to%20add%20test%20for%20all%20your%20components%20and%20functions.).

Reply:
@gemmen29 Thanks!! Done.